neps/nep-0393.md

```diff
@@ -425,98 +537,12 @@ trait SBT {
-```
-
-#### Mint tokens
-
-Alice uses her `alice1` account to interact with `SBT_1 Issuer` and receives an SBT with token ID = 238:
-
-```mermaid
-sequenceDiagram
-    actor Alice
-    actor Issuer1 as SBT_1 Issuer
-
-    participant SBT1 as SBT_1 Contract
-    participant SBT_Registry
-
-    Issuer1->>SBT1: sbt_mint(alice1, metadata)
-    activate SBT1
-    SBT1-)SBT_Registry: sbt_mint([[alice1, [metadata]]])
-    SBT_Registry->>SBT_Registry: emit Mint(SBT_1_Contract, alice1, [238])
-    SBT_Registry-)SBT1: [238]
-    deactivate SBT1
-
-    Note over Alice,SBT_Registry: now Alice can query registry to check her SBT
-
-    Alice-->>SBT_Registry: sbt(SBT_1_Contract, 238)
-    SBT_Registry-->>Alice: {token: 238, owner: alice1, metadata}
-```
-
-With `SBT_2 Issuer`, Alice uses her `alice2` account. Note that `SBT_2 Contract` has different mint function (can mint many tokens at once), and validates a proof prior to requesting the registry to mint the tokens.
-
-```mermaid
-sequenceDiagram
-    actor Alice
-    actor Issuer2 as SBT_2 Issuer
-
-    participant SBT2 as SBT_2 Contract
-    participant SBT_Registry
-
-    Issuer2->>SBT2: sbt_mint_multi([[alice2, metadata2], [alice2, metadata3]], proof)
-    activate SBT2
-    SBT2-)SBT_Registry: sbt_mint([[alice2, [metadata2, metadata3]]])
-    SBT_Registry->>SBT_Registry: emit Mint(SBT_2_Contract, alice2, [7991, 7992])
-    SBT_Registry-)SBT2: [7991, 1992]
-    deactivate SBT2
-
-    Note over Alice,SBT_Registry: Alice queries one of her new tokens
-    Alice-->>SBT_Registry: sbt(SBT_2_Contract, 7991)
-    SBT_Registry-->>Alice: {token: 7991, owner: alice2, metadata: metadata2}
-```
-
-#### Soul transfer
-
-After a while Alice decides to transfer her `alice2` soul into `alice1`
-
-```mermaid
-sequenceDiagram
-    actor Alice
-    participant SBT_Registry
-
-    Alice->>SBT_Registry: sbt_soul_transfer(alice1) --accountId alice2
-
-    Alice-->>+SBT_Registry: sbt_tokens_by_owner(alice2)
-    SBT_Registry-->>-Alice: []
-
-    Alice-->>+SBT_Registry: sbt_tokens_by_owner(alice1)
-    SBT_Registry-->>-Alice: [[SBT_1_Contract, [238]], [SBT_2_Contract, [7991, 7992]]]}
-```
-
 ## Consequences
 
 Being fully compatible with NFT standard is a desirable. However, given the requirements related to _soul transfer_ we didn't find an applaudable solution. Also we decided to use u64 as a Token ID, diverging further from the NFT NEP-171 standard.
 
 Given that our requirements are much striker, we had to reconsider the level of compatibility with NEP-171 NFT.
-There are many examples where NFT standards are poorly or improperly implemented, adding another standard with different functionality but equal naming will cause lots of problems and misclassifications between NFT and SBT.
+There are many examples where NFT standards are improperly implemented, adding another standard with different functionality but equal naming will cause lots of problems and misclassifications between NFT and SBT.
```

Comment:
Make it via dot not a comma like this: "...implemented. Adding another standard..." as I realized I misread the sentence thrice.